simple.js

```diff
@@ -1381,19 +1378,22 @@ define(function(require, exports, module) {
                 // set Python's version
                 settings.set("project/python/@version", "python3");
 
-                // set pylint's flags
-                settings.set(
-                    "project/python/@pylintFlags",
-                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQLAlchemy --load-plugins pylint50"
-                );
-
                 // set PYTHONPATH
                 settings.set(
                     "project/python/@path",
                     "/home/ubuntu/.local/lib/python3.4/site-packages" +
                     ":/usr/lib/python3/dist-packages" +
                     ":/usr/local/lib/python3.4/dist-packages"
                 );
+
+                // set pylint's flags
+                settings.set(
+                    "project/python/@pylintFlags",
+                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQL,SQLAlchemy --load-plugins pylint50"
```

Comment:
If it makes any difference, Cloud9 keep some stuff there as well (e.g., `~/.c9/`). I'd say dot files in home directory isn't that uncommon. And technically, the project directory for CS50 workspaces is `~/workspace/`, so users should be spending most of their time in that directory. Can move somewhere else though, if you'd like.